RealEstate/apps/pending/models.py

```diff
@@ -18,7 +18,8 @@ def _generate_registration_token():
     SHA-256.
     """
     while True:
-        token = hashlib.sha256(get_random_string(length=64)).hexdigest()
+        token = hashlib.sha256(
+            get_random_string(length=64).encode(encoding='utf-8')).hexdigest()
```

Comment:
Dont need to utf8 encode the string, the default character set is alphanumeric only.